index.js

```diff
@@ -22,83 +22,407 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
-import sh from 'execSync';
+//import sh from 'execSync';
+import i2c from 'i2c-bus';
+import { execSync } from 'child_process';
 import { Peripheral } from 'raspi-peripheral';
+import { VERSION_1_MODEL_B_REV_1, getBoardRevision } from 'raspi-board';
+
+if (typeof execSync !== 'function') {
+  execSync = require('execsync');
+}
+
+function checkAlive(alive) {
```

Comment:
The coding style for the rest of Raspi.js is to put these functions as methods in the class, not as stand-alone functions. Can you move them? This also would negate the need to pass the alive value into the function